.storybook/webpack.config.js

```diff
@@ -55,9 +55,22 @@ module.exports = async ( { config } ) => {
 		},
 	);
 
+	// exclude existing svg rule created by storybook before pushing custom rule
+	const fileLoaderRule = config.module.rules.find( ( rule ) => rule.test && rule.test.test( '.svg' ) );
+	fileLoaderRule.exclude = /\.svg$/;
+
+	config.module.rules.push( {
+		test: /\.svg$/,
+		loader: '@svgr/webpack',
+		options: {
+			// strip width & height to allow manual override using props
+			dimensions: false,
+		},
+	} );
+
 	config.module.rules.push(
 		{
-			test: /\.(png|woff|woff2|eot|ttf|svg|gif)$/,
+			test: /\.(png|woff|woff2|eot|ttf|gif)$/,
```

Comment:
Maybe we should export these in a way that we can use them from the main webpack config file rather than duplicating the rules? We already share a few things with it.